go.mod

```diff
@@ -809,12 +809,10 @@ require (
 	k8s.io/kube-openapi v0.0.0-20240620174524-b456828f718b // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.4.1 // indirect
-
-)
-
-require (
+	github.com/gammazero/deque v0.2.1 // indirect
+	github.com/vladopajic/go-actor v0.9.0 // indirect
 	github.com/aws/aws-sdk-go-v2/service/iam v1.33.1 // indirect
-	github.com/checkpoint-restore/go-criu/v6 v6.3.0 // indirect
+    github.com/checkpoint-restore/go-criu/v6 v6.3.0 // indirect
```

Comment:
nit: indentation

Reply:
Tabs vs spaces! But fixed so its consistent. I was confused for a bit when opened it up in my editor.